.github/workflows/default.yml

```diff
@@ -106,6 +106,7 @@ jobs:
         run: |
          eval $(opam env)
          make tests
+	 ./run-tests nodb-unit-tests
```

Comment:
I think YAML insists that you line up L107-109 on the same column.

Reply:
You also need to install `ounit2` in the job, e.g. on line https://github.com/links-lang/links/blob/3ba9c18bd77dc6ec7aa1d655d5ef404308c07948/.github/workflows/default.yml#L91